Formula/flock.rb

```diff
@@ -0,0 +1,25 @@
+class Flock < Formula
+  desc "Lock file during command"
+  homepage "https://github.com/discoteq/flock"
+  url "https://github.com/discoteq/flock/releases/download/v0.4.0/flock-0.4.0.tar.xz"
+  sha256 "01bbd497d168e9b7306f06794c57602da0f61ebd463a3210d63c1d8a0513c5cc"
+  license "ISC"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    pid = fork do
+      exec bin/"flock", "tmpfile", "sleep", "5"
+    end
+    sleep 1
+    shell_output("#{bin}/flock --nonblock tmpfile true", 1)
```

Comment:
```suggestion
    assert_match "something", shell_output("#{bin}/flock --nonblock tmpfile true", 1)
```

Reply:
I'm now asserting that the command output is empty